src/HMR/babelPlugin.js

```diff
@@ -0,0 +1,149 @@
+/**
+ * Copyright 2017-present, Callstack.
+ * All rights reserved.
+ */
+
+/**
+ * Ideally, this file SHOULD NOT be processed by prettier, since trailing commas
+ * are not supported and will cause errors in bundle. This file is used by `babel-loader`
```

Comment:
My personal opinion is yes, because not all of the code we write will be transpiled. So it'll be nice to have an eslint config which works for every project.